recipe/run_test.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pkgutil
+import platform
+import sys
+import subprocess
+
+import pytest
+from prophet import Prophet
+
+try:
+    import prophet.tests
+except ImportError:
+    print('Error: Unable to import prophet.tests')
+    sys.exit(1)
+
+def go():
+    py_major = sys.version_info[0]
+    py_impl = platform.python_implementation().lower()
+    machine = platform.machine().lower()
+
+    print("Python implementation:", py_impl)
+    print("              Machine:", machine)
+
+    m = Prophet()
```

Comment:
Just wondering why the need for the `run_test.py`? E.g. in the recipe you could do
`python -c "from prophet import Prophet; assert Prophet.stan_backend.get_type() = "cmdstan"` or similar
If this is a lot of work don't worry about it, it's really just a preference, better to get it in before sprint end

Reply:
It was my initial suggestion for testing it more deeply but, yeah, it can be simplified now.